### PR TITLE
DBZ-8486 Add Flyway to manage schema evolution

### PR DESCRIPTION
**Pull request by mfvitale**
closes: https://issues.redhat.com/browse/DBZ-8486

### CODE REVIEW

**Comment by jcechace** [comment]
@mfvitale  just a small note regarding constrain naming, otherwise I think this can go in.

File: debezium-platform-conductor/src/main/resources/db/migration/V0.1.0__initial_database.sql [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    plaintext boolean not null,
+    id bigint not null,
+    description varchar(255),
+    name varchar(255) not null unique,
+    items jsonb,
+    primary key (id)
+);
+
+alter table if exists destination_vault
+   add constraint FK3iovoim04er4ryyxc3ahg30aj
```
[the remark this answers is not in the document]
Comment (mfvitale) [reply debezium-platform-conductor/src/main/resources/db/migration/V0.1.0__initial_database.sql:97]: Yeah, you caught me ;-). I'll rename with a better name.

[the remark this answers is not in the document]
Comment (jcechace) [reply debezium-platform-conductor/src/main/resources/db/migration/V0.1.0__initial_database.sql]: Well, it is a great tool for the initial migration :).

**Review by jcechace — APPROVED** [review]

**Comment by jcechace** [comment]
LGTM -- feel free to merge, unless you need Jiri to also take a look.

**Pull request merged**